src/controllers/customercontroller.js

```diff
@@ -6,29 +6,81 @@ class CustomerController extends OrderController
             "src/views/template/customer.html",
             "#customers",
             {}
-        );
+        ).done(() => this.registerEvent());
+    }
+
+    registerEvent()
+    {
+        $(document).on('click', '.customer-complete', function(e) {
+            e.preventDefault();
+
+            var id = e.target.parentNode.parentNode.id.split('-')[1];
+
+            let customerController = new CustomerController();
+            customerController.completeOrder(id);
+        });
     }
 
     generateOrder()
     {
+        if (super.normalDistribution() < 1.725) {
```

Comment:
I'm not sure where the order is generated exactly (i.e., where is this function invoked from?), but perhaps this check should be instated there, and not done in the `generateOrder` function.